js-miniapp-sdk/src/types/adTypes/index.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Supported ad formats
+ */
+export enum AdTypes {
+  BANNER,
```

Comment:
We currently don't plan to support banner ads, so I think this should be removed.

Reply:
+1